examples/update/step2/index.ts

```diff
@@ -8,6 +9,10 @@ const logGroup = new aws.s3.Bucket("bucket", {
       key: "foo",
       value: "buzz", // <-- this value has changed
     },
+    {
+      key: "secretfoo",
+      value: pulumi.secret("secretbuzz"), // <-- this value has changed
+    },
```

Comment:
How do we assert that this value has been updated and not missed?

Reply:
Good question, currently we don't. The failure was at the time of serialization, so we prove we don't get that error. Let me see if we have prior art on update validation.

Reply:
Done now: exported the secret value and validated it from the test.